tests/testthat/tests-clone_instat_calculation.R

```diff
@@ -32,7 +10,7 @@ test_that("clone_instat_calculation clones a basic instat_calculation", {
     save = TRUE
   )
   
-  cloner <- Cloner$new()
```

Comment:
Simply, we use our `DataBook` here. That is the function we want to test, as that is the function that users have access to